File: MORM.Java/src/MORM/Java/Models/Venda/Transdfe.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TRANSDFE")
public class Transdfe extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TRANSACAO", Tipo = CampoTipo.Key)
    public String Id_Transacao;
    @CampoAttribute(Campo = "NR_SEQUENCIA", Tipo = CampoTipo.Key)
    public Integer Nr_Sequencia;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "TP_EVENTO", Tipo = CampoTipo.Req)
    public Integer Tp_Evento;
    @CampoAttribute(Campo = "TP_AMBIENTE", Tipo = CampoTipo.Req)
    public Integer Tp_Ambiente;
    @CampoAttribute(Campo = "TP_EMISSAO", Tipo = CampoTipo.Req)
    public Integer Tp_Emissao;
    @CampoAttribute(Campo = "DS_ENVIOXML", Tipo = CampoTipo.Req)
    public String Ds_Envioxml;
    @CampoAttribute(Campo = "DS_RETORNOXML", Tipo = CampoTipo.Nul)
    public String Ds_Retornoxml;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Transdfe.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSDFE")]
    public class Transdfe : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.Key)]
        public string Id_Transacao { get; set; }
        [Campo("NR_SEQUENCIA", CampoTipo.Key)]
        public int Nr_Sequencia { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("TP_EVENTO", CampoTipo.Req)]
        public int Tp_Evento { get; set; }
        [Campo("TP_AMBIENTE", CampoTipo.Req)]
        public int Tp_Ambiente { get; set; }
        [Campo("TP_EMISSAO", CampoTipo.Req)]
        public int Tp_Emissao { get; set; }
        [Campo("DS_ENVIOXML", CampoTipo.Req)]
        public string Ds_Envioxml { get; set; }
        [Campo("DS_RETORNOXML", CampoTipo.Nul)]
        public string Ds_Retornoxml { get; set; }
    }
}